FirebaseAnalyticsSwift/Tests/ObjCAPI/ObjCAPITests.m

```diff
@@ -50,13 +50,16 @@ - (void)appDelegateTests:(NSURL *)url {
 
 - (void)consentTests:(NSURL *)url {
   [FIRAnalytics setConsent:@{
-    FIRConsentTypeAnalyticsStorage : FIRConsentStatusGranted,
-    FIRConsentTypeAdStorage : FIRConsentStatusDenied
+    FIRConsentTypeAdPersonalization : FIRConsentStatusGranted,
+    FIRConsentTypeAdStorage : FIRConsentStatusDenied,
+    FIRConsentTypeAdUserData : FIRConsentStatusGranted,
+    FIRConsentTypeAnalyticsStorage : FIRConsentStatusDenied,
```

Comment:
Similar to my other comment, it'd be ideal to have some switch statements that would fail to compile should the cases change. I would prioritize my other comment though if this proves infeasible.

Reply:
Thanks for the suggestion. Unfortunately they are just NSString values in ObjC. I cannot use switch statement to check for completeness.